libs/client-shared/src/lib/components/chip/chip.component.html

```diff
@@ -0,0 +1,3 @@
+<mat-chip [disabled]="true"
```

Comment:
Since this is always set, you can simplify it.
```suggestion
<mat-chip disabled
```

Reply:
when i remove it, the ide copmlains: 
`disabled requires value `

Reply:
alright, that's just material being silly then. Resolved.